pg_analytics/src/datafusion/plan.rs

```diff
@@ -0,0 +1,22 @@
+use deltalake::datafusion::error::DataFusionError;
+use deltalake::datafusion::logical_expr::LogicalPlan;
+use deltalake::datafusion::sql::parser::DFParser;
+use deltalake::datafusion::sql::planner::SqlToRel;
+use deltalake::datafusion::sql::sqlparser::dialect::PostgreSqlDialect;
+
+use crate::datafusion::context::QueryContext;
+use crate::errors::ParadeError;
+
+#[inline]
+pub fn create_logical_plan(query: &str) -> Result<LogicalPlan, ParadeError> {
```

Comment:
Duplicate implementation of `hooks/query.rs`

Reply:
I'm going to refactor this. The whole structure of having `Query` trait implemented over `PlannedStmt` restricts the use cases for `get_ast` and `get_logical_plan` when they don't even depend on `PlannedStmt`. They're simply utility functions over a SQL string.